qemu/tests/cfg/secure_img.cfg

```diff
@@ -0,0 +1,17 @@
+- secure_img:
+    only s390x
+    type = secure_img
+    take_regular_screendumps = no
+    start_vm = yes
+    kill_vm_before_test = yes
+    kill_vm = yes
+    machine_type_extra_params = "usb=off,dump-guest-core=off,confidential-guest-support=lsec0"
+    extra_params = "-object s390-pv-guest,id=lsec0"
+    nic_extra_params = "devno=fe.0.0001"
+    virtio_serial_extra_params = "devno=fe.0.0002"
+    secure_params = 'echo "$(cat /proc/cmdline) swiotlb=262144" > /home/parmfile'
+    # For HKD.crt in boot_img, it's a private cerfitication for specific s390x machines
+    boot_img = 'genprotimg -k /home/HKD.crt -p /home/parmfile -i %s -r %s -o /boot/secure-linux --no-verify'
+    zipl_config = 'echo -e "[secure]\ntarget=/boot\nimage=/boot/secure-linux">> /etc/zipl.conf'
+    zipl = 'zipl'
+    check_se = 'cat /sys/firmware/uv/prot_virt_guest'
```

Comment:
Why do you use parameters for commands? If you do so, I would consider different names to avoid confusions. For example, `secure_params` sounds like a parameter that contains the kernel command parameters for the secure image, but it's actually a command that writes those parameters into a file in the /home directory. Similarly, `boot_img`, `zipl_config`, `zipl` and `check_se`.

Reply:
Updated